geth/node/txqueue_manager.go

```diff
@@ -162,8 +162,10 @@ func (m *TxQueueManager) CompleteTransaction(id common.QueuedTxID, password stri
 	var txErr error
 
 	if config.UpstreamConfig.Enabled {
+		log.Info("completing completed upstream", "id", queuedTx.ID)
```

Comment:
There is a log inside `completeRemoteTransaction`.

Reply:
Thanks, updated!